src/js/core.js

```diff
@@ -899,7 +899,7 @@
 
             try {
                 this.events.disableCustomEvent('editableInput');
-                if (opts.url && opts.url.trim().length > 0) {
+                if (opts.url && opts.url.length > 0) {
```

Comment:
Why would we want to allow for the creation of a link with a url which is only whitespace here?